src/Batch/Batch/help/Set-AzBatchPoolOSVersion.md

```diff
@@ -1,116 +0,0 @@
-﻿---
```

Comment:
@matthchr is this cmdlet being removed? If so, we'll want to remove it from the `.psd1` file so it's not being exported

Reply:
Oh, good catch. Yes it is being removed.

Reply:
I've fixed the `psd1` file now